content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -130,31 +87,33 @@ can set the number with `--jobs` (`-j`).
 If an experiment being pulled already exists in the local project, DVC won't
 overwrite it unless you supply `--force`.
 
-### Example: Pushing or pulling multiple experiments
+## Example: Sharing multiple experiments
 
-You can create a loop to upload or download all experiments like this:
+You can create a loop to push or pull all experiments. For example in a Linux
+terminal:
 
 ```dvc
 $ dvc exp list --all --names-only | while read -r expname ; do \
     dvc exp pull origin ${expname} \
 done
 ```
 
-> Without `--all`, only the experiments derived from the current commit will be
-> pushed/pulled.
+> 📖 See also [Listing experiments].
 
-## Example: Creating a directory for an experiment
+[listing experiments]:
+  /doc/user-guide/experiment-management/comparing-experiments#list-experiments-in-the-project
 
-A good way to isolate experiments is to create a separate home directory for
-each one.
+## Example: Dedicated experiment directories
```

Comment:
I don't quite understand where does this example come from, what is the motivation ... it also feels very unrelated to the section itself

Reply:
For example, the user has experimented for some time and found a good result. This is to isolate that experiment in a single directory and share with folks that don't use Git or DVC.

Reply:
I don't have a strong opinion on this example. I think it's clever but also agree it's not clear whether someone would ever do this. Keeping it for now as it's at the bottom but I'm happy to remove it if you guys decide ( we already have other examples both here and in https://dvc.org/doc/command-reference/exp/push#examples).